database/migrations/2020_08_03_170501_atualiza_nomenclatura_dos_campos_da_tela_de_configuracoes.php

```diff
@@ -29,14 +29,16 @@ public function up()
                     WHEN \'legacy.app.alunos.nao_apresentar_campo_alfabetizado\' THEN \'Não apresentar check "Alfabetizado" no cadastro do aluno\'
                     WHEN \'legacy.app.alunos.obrigar_recursos_tecnologicos\' THEN \'Obrigar informar recursos tecnológicos do aluno\'
                     WHEN \'legacy.app.auditoria.notas\' THEN \'Auditar lançamento de notas\'
-                    WHEN \'legacy.app.aws.bucketname\' THEN \'Bucket S3 para armazenar uploads\'
-                    WHEN \'legacy.app.database.dbname\' THEN \'Nome\'
+                    WHEN \'legacy.app.aws.bucketname\' THEN \'AWS Bucket S3 para armazenar uploads\'
+                    WHEN \'legacy.app.aws.awsacesskey\' THEN \'AWS Bucket S3 Acess Key\'
+                    WHEN \'legacy.app.aws.awssecretkey\' THEN \'AWS Bucket S3 Secret Key\'
+                    WHEN \'legacy.app.database.dbname\' THEN \'Nome da database\'
                     WHEN \'legacy.app.database.hostname\' THEN \'Host\'
                     WHEN \'legacy.app.database.password\' THEN \'Senha\'
                     WHEN \'legacy.app.database.port\' THEN \'Porta\'
                     WHEN \'legacy.app.database.username\' THEN \'Usuário\'
                     WHEN \'legacy.app.diario.nomenclatura_exame\' THEN \'Apresentar nota de exame como nota de conselho\'
-                    WHEN \'legacy.app.entity.name\' THEN \'Nome da instituição\'
+                    WHEN \'legacy.app.entity.name\' THEN \'Nome da entidade\'
```

Comment:
Mantém o original porque é usado o termo "Instituição" dentro do i-Educar

```suggestion
                    WHEN \'legacy.app.entity.name\' THEN \'Nome da instituição\'
```

Reply:
Ajuste realizado @edersoares